style.css

```diff
@@ -0,0 +1,104 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --color-primary-text-1: #ff6a01;
+  --color-supporting-text-4: #fef5e0;
+  --color-icons-8: #dbd8d7;
+  --color-description-text-11: #fff;
+  --color-background-1: #1c1a19;
+  --color-nav-bar: #3c3a39;
+}
+
+.header {
+  width: 375px;
+  height: 48px;
+  background-color: var(--color-nav-bar, #3d3b39);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px 24px 2px 12px;
+}
+
+.header .logo {
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  color: var(--color-icons-8);
+  text-decoration: none;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+.hero {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 375px;
+  height: 98vh;
```

Comment:
- [ ] Please don't use the `height` property in your design because you are doing the same thing that I mentioned in my above `inline-comment`. Instead with the help of `padding` you can let make children of the header decide the `height` of the header because it will be more flexible and you can easily customize your element's size.